site2/docs/concepts-messaging.md

```diff
@@ -263,14 +264,15 @@ The message redelivery behavior should be as follows.
 
 Redelivery count | Redelivery delay
 :--------------------|:-----------
-1 | 10 + 1 seconds
```

Comment:
The 2nd set of examples are for the ack timeout redelviery delivery delay. It makes sense that the overall delay is timeout (10s) + backoff). But here the examples are for the negative ack redelivery delay which should just be the backoff time from the negative ack if I understand correctly. Hence why I've removed the `10 +`.

Reply:
@tjiuming any feedback on this?